frontend/src/hooks/api/getters/useChangeRequestConfig/useChangeRequestConfig.ts

```diff
@@ -8,7 +8,11 @@ export const useChangeRequestConfig = (projectId: string) => {
         IChangeRequestEnvironmentConfig[]
     >(
         [],
-        formatApiPath(`api/admin/projects/${projectId}/change-requests/config`),
+        Boolean(projectId)
+            ? formatApiPath(
+                  `api/admin/projects/${projectId}/change-requests/config`
+              )
+            : null,
```

Comment:
I'm unsure if this would be the best approach based on what @kwasniew has shared in the past.

How about using a `useConditionalSWR` instead?

```TS
const { isEnterprise } = useUiConfig();

const { data, error, mutate } = useConditionalSWR<
        IChangeRequestEnvironmentConfig[]
    >(
        Boolean(projectId) && isEnterprise(),
        [],
        formatApiPath(`api/admin/projects/${projectId}/change-requests/config`),
        fetcher
    );
```

Reply:
This seems to be the recommended way based on https://swr.vercel.app/docs/conditional-fetching

Happy to change to the useConditionalSWR if you like that better

Reply:
@kwasniew thoughts?

Reply:
+1 for useConditionalSWR to be consistent with all our code